modules/installation-creating-gcp-bootstrap.adoc

```diff
@@ -0,0 +1,106 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_gcp_user_infra/installing-gcp-user-infra.adoc
+
+[id="installation-creating-gcp-bootstrap_{context}"]
+= Creating the bootstrap machine in GCP
+
+You must create the bootstrap machine in Google Cloud Platform (GCP) to use during
+{product-title} cluster initialization. The easiest way to create this machine is
+to modify the provided Deployment Manager template.
```

Comment:
"The easiest" -> "One"?  I dunno what the other ways are, but readers might have an easier way via their internal tooling or whatever.  I don't think we need to rank options by ease ;).